.dockerignore

```diff
@@ -1,13 +1,12 @@
 # config
-env.json
+**/env.*
 venv
 
 # data
-*.csv
+**/*.csv
 .data
 
 # etc
 .DS_Store
-*.pyc
-.vscode
```

Comment:
I don't think there's any need to include this in the image but there's no harm in it either. You'd have to be running vscode inside the container for this to matter.

Reply:
I guess I was thinking about the debugger (which I still don't have a lot of experience with). You mentioned at one point that you can use this OpenShift. Would you not need `.vscode` to do that? I can add this back if it's not needed.

Reply:
Okay, that makes sense. I'll add it back to the file.